### ISSUE TITLE
Allow returning to one's own locked snippet

### ISSUE THREAD

**Issue by L2G** [fanscribed/fanscribed#38]
I've had a couple of times where a bad keystroke dumped me out of my transcript. (That will be a new issue once I've researched the circumstances a little more.) :-)

A nice feature would be to show any snippets currently locked for the user and allow the user to get back into them. (This may need to be contingent on getting authentication implemented, though?)
